invenio_users_resources/records/api.py

```diff
@@ -37,6 +37,9 @@ def parse_user_data(user):
 
     data["preferences"].setdefault("visibility", "restricted")
     data["preferences"].setdefault("email_visibility", "restricted")
+    data["preferences"].setdefault("locale", "en_GB")
```

Comment:
What if instead we use the config vars here?
For example, if one instance has defined `BABEL_DEFAULT_LOCALE = "it_IT"`, you want to do:
```suggestion
    default_locale = current_app.config("BABEL_DEFAULT_LOCALE", "en_GB")
    data["preferences"].setdefault("locale", default_locale)
```
Same for the timezone.